envs/monkey_zoo/blackbox/utils/gcp_machine_handlers.py

```diff
@@ -43,26 +40,31 @@ def get_absolute_key_path() -> str:
             )
         return absolute_key_path
 
-    def start_machines(self, machine_list):
+    @staticmethod
```

Comment:
Since this class no longer maintains any state and only contains static methods, it doesn't need to be a class. We can just create an `initialize_gcp_client()` function that performs the same actions as the constructor and call that before the start/stop machine functions.